index.ts

```diff
@@ -0,0 +1,20 @@
+/** Modules **/
+
+export { flip } from "./src/flip";
+export { log } from "./src/log";
+export { print } from "./src/print";
+export { random } from "./src/random";
+export { uuid } from "./src/uuid";
+
+/** Converting Modules **/
```

Comment:
Please move this section into util section as these are no longer located in the types module